src/Modules/Workflows/Controllers/WorkflowsList.php

```diff
@@ -73,7 +73,7 @@ public function initialize()
         );
 
         $this->hooks->addAction(
-            "post_row_actions",
+            HooksAbstract::FILTER_POST_ROW_ACTIONS,
```

Comment:
@ojopaul this is an "action", but was named as "filter"

Reply:
Just correcting my comment, the issue is actualy in the line before. It should be `$this->hooks->addFilter`.